ee/clickhouse/views/person.py

```diff
@@ -16,6 +19,20 @@ class ClickhousePersonViewSet(PersonViewSet):
     retention_class = ClickhouseRetention
     stickiness_class = ClickhouseStickiness
 
+    @action(methods=["GET"], detail=False)
+    def funnel(self, request: request.Request, **kwargs) -> response.Response:
+        filter = Filter(request=request)
+        team = request.user.team
+        results = ClickhouseFunnelPersons(filter, team).run()
+        return response.Response(data=results)
```

Comment:
one last thing is that for any paginated API endpoint, we generally pass back a formatted URL that can be easily called by the frontend[ like this](https://github.com/PostHog/posthog/blob/master/posthog/api/person.py#L240). The pass backed url also notifies the frontend that there are more results